### ISSUE TITLE
[Bug]: fdt build throws exception when user logs with format other than %s (e.g. %d)

### ISSUE THREAD

**Issue by bernhardschaefer** [emdgroup/foundry-dev-tools#41]
### Issue checklist

- [X] This is a **bug** in Foundry DevTools and not a bug in another project. It is also not an enhancement/feature request
- [X] I searched through the GitHub issues and this issue has not been opened before.
- [X] I have installed the latest version of Foundry DevTools and don't use an unsupported python version.

### Description of the bug

I have custom logging statements in a transform and fdt build throws an error (see below) when these log statements contain %d formatters.
I think the issue is in [build.py#L75](https://github.com/emdgroup/foundry-dev-tools/blob/main/src/foundry_dev_tools/cli/build.py#L75) where all logging params are converted to string.

### Steps to reproduce this bug.

Should be sufficient to create a transform with a log statement such as `logging.info("hello %d", 3)`

### Log output

```shell
[08:37:14] ERROR    fdt build >>> This shouldn't happen, but while parsing the log message this error occured: %d format: a real number is required, not str              build.py:152
                                                                                                                                                                                      
           ERROR    The traceback:                                                                                                                                        build.py:156
                    Traceback (most recent call last):                                                                                                                                
                      File "/opt/homebrew/Caskroom/mambaforge/base/envs/myenv/lib/python3.11/site-packages/foundry_dev_tools/cli/build.py", line 150, in                     
                    tail_job_log                                                                                                                                                      
                        log.handle(create_log_record(log_message))                                                                                                                    
                      File "/opt/homebrew/Caskroom/mambaforge/base/envs/myenv/lib/python3.11/logging/__init__.py", line 1644, in handle                                      
                        self.callHandlers(record)                                                                                                                                     
                      File "/opt/homebrew/Caskroom/mambaforge/base/envs/myenv/lib/python3.11/logging/__init__.py", line 1706, in callHandlers                                
                        hdlr.handle(record)                                                                                                                                           
                      File "/opt/homebrew/Caskroom/mambaforge/base/envs/myenv/lib/python3.11/logging/__init__.py", line 978, in handle                                       
                        self.emit(record)                                                                                                                                             
                      File "/opt/homebrew/Caskroom/mambaforge/base/envs/myenv/lib/python3.11/site-packages/rich/logging.py", line 128, in emit                               
                        message = self.format(record)                                                                                                                                 
                                  ^^^^^^^^^^^^^^^^^^^                                                                                                                                 
                      File "/opt/homebrew/Caskroom/mambaforge/base/envs/myenv/lib/python3.11/logging/__init__.py", line 953, in format                                       
                        return fmt.format(record)                                                                                                                                     
                               ^^^^^^^^^^^^^^^^^^                                                                                                                                     
                      File "/opt/homebrew/Caskroom/mambaforge/base/envs/myenv/lib/python3.11/logging/__init__.py", line 687, in format                                       
                        record.message = record.getMessage()                                                                                                                          
                                         ^^^^^^^^^^^^^^^^^^^                                                                                                                          
                      File "/opt/homebrew/Caskroom/mambaforge/base/envs/myenv/lib/python3.11/logging/__init__.py", line 377, in getMessage                                   
                        msg = msg % self.args                                                                                                                                         
                              ~~~~^~~~~~~~~~~                                                                                                                                         
                    TypeError: %d format: a real number is required, not str                                                                                                          
           INFO     fdt build >>> Will output the log message in plain:                                                                                                   build.py:157
                                                                                                                                                                                      
           INFO     {"type": "service.1", "level": "INFO", "origin": "python:root", "thread": "Thread-3", "message": "Retrieved %d items for title %s", "time":           build.py:160
                    "2023-11-29T07:37:14.231Z", "tags": {"jobRid": "ri.foundry.main.job.asdf"}, "params": {}, "unsafeParams": {"param_0":             
                    514, "param_1": "My Title"}}                                                                                                        
           ERROR    fdt build >>> This shouldn't happen, but while parsing the log message this error occured: %d format: a real number is required, not str              build.py:152
```

### Operating System

MacOS

### Your python version

Python 3.11.6

**Comment by jonas-w** [comment]
Hey, thank you for reporting! Can you test out the PR #42 with your build?

**Comment by bernhardschaefer** [comment]
Just tested it, seems to work now, thanks for implementing the fix. :-)
```shell
           INFO     Retrieved 514 items for title xyz                                python:root
           INFO     Filtered to 349 items                                                   python:root
```

**Comment by jonas-w** [comment]
closed in [v1.3.4](https://github.com/emdgroup/foundry-dev-tools/releases/tag/v1.3.4)
